modules/executable-dependency/main.tf

```diff
@@ -3,6 +3,13 @@ terraform {
   # 0.12.26 as the minimum version, as that version added support for required_providers with source URLs, making it
   # forwards compatible with 0.13.x code.
   required_version = ">= 0.12.26"
+
+  required_providers {
+    external = {
+      source  = "hashicorp/external"
+      version = "2.0.0"
```

Comment:
Do we want to be pinned to an _exact_ version here?

Reply:
Same question for all the other providers pinned to a specific version, rather than, say, `~> 2.0.0`.